src/reporting/config.zig

```diff
@@ -72,7 +73,15 @@ pub const ReportingConfig = struct {
         };
 
         // Check if output is TTY
-        config.is_tty = std.io.getStdOut().isTty();
+        config.is_tty = isTty: {
+            if (comptime builtin.target.os.tag != .freestanding) {
+                break :isTty false;
```

Comment:
Is this necessary to change the non-freestanding case?  I understand for WASM but changing this for other targets would break a potential CLI REPL.  Also this entire assignment is really just `false`

Reply:
I'm not sure we need this. When we turned on the freestanding WASM target zig blows up with a cryptic error about stdio. So I went on a crusade and just had to turn everything that could be related off -- now we have a working baseline I can go back through these and see if it is actually needed.